staging/src/k8s.io/apiserver/pkg/registry/generic/registry/storage_factory.go

```diff
@@ -88,6 +90,17 @@ func StorageWithCacher(capacity int) generic.StorageDecorator {
 	}
 }
 
+func objectTypeToString(obj runtime.Object) string {
```

Comment:
In a language with runtime representations for types, such as go, it is actually confusing to use the name "object type" instead of "example object" when the latter is meant.  (This complaint applies to a lot of code --- particularly code that uses the golang type `interface{}` for the value in question --- not just this change.)